app/src/main/res/layout/fragment_registration.xml

```diff
@@ -163,13 +163,13 @@
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:digits="0123456789"
+                    android:inputType="numberPassword"
                     android:background="@drawable/white_rounded_background"
                     android:hint="@string/student_id"
-                    android:imeOptions="actionDone"
+                    android:imeOptions="flagNavigateNext"
```

Comment:
This parameter is ignored on my device, but it works correctly in the authorization fragment. The first image shows an option that, when you click on the button, moves to the next field, but not in the second one. Please check it again on your device and give feedback
![photo_2023-12-20_16-54-02](https://github.com/NN171/Head-of-the-group-helper/assets/129608390/ee35603f-89c3-4427-adcc-fe93172face3)
![photo_2023-12-20_16-54-07](https://github.com/NN171/Head-of-the-group-helper/assets/129608390/4df6965f-879c-4b99-a2f0-4dd12f545c5a)

Reply:
Yes, in my previous comment, the buttons are also different

Reply:
I fixed it, check please